README.md

```diff
@@ -1417,6 +1418,55 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /api/workflows/:version/query
```

Comment:
Might be good to know which of the keys are accepted as GET parameters and what they mean